### PR TITLE
Decouple MBI from Advanced Reports (DevDocs side)

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR)  addresses task MBI-11185. Need to decouple MBI from advanced reporting as it was causing confusion among customers who thought you needed MBI to get advanced reporting to work.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/advanced-reporting/overview.html
- https://devdocs.magento.com/guides/v2.4/advanced-reporting/data-collection.html
- https://devdocs.magento.com/guides/v2.4/advanced-reporting/modules.html

### CODE REVIEW

File: src/guides/v2.3/advanced-reporting/overview.md [hunk 1]
```diff
@@ -5,8 +5,7 @@ functional_areas:
     - Reports
 ---
 
-[Advanced reporting functionality]{:target="_blank"} is provided through integration of a Magento instance with the [Magento Business Intelligence]{:target="_blank"} (MBI).
-Magento collects data and sends this information to the MBI for analytics.
+[Advanced reporting functionality]{:target="_blank"} is free to Magento Commerce and Open Source customers and is provided through an integration of a Magento instance with [Magento Business Intelligence]{:target="_blank"} (MBI). Magento collects data and sends this information to MBI for analytics. You do not need to have an MBI account to use Advanced Reporting.
```
Comment (keharper) [inline src/guides/v2.3/advanced-reporting/overview.md:8]: Global in this doc: remove the `{:target="_blank"}`  directives.

File: src/guides/v2.3/advanced-reporting/overview.md [hunk 2]
```diff
@@ -17,18 +16,18 @@ Magento collects data and sends this information to the MBI for analytics.
...
+It can take up to a day for data to be available in Advanced Reporting.
 
 ## Recommendations
 
 To avoid system overload during its prime time, you can set the preferable time of a day for a data collection.
 
 ## Extensibility
 
-Though the Analytics module provides an API, it is used specifically to interchange data with the MBI. We do not recommend to extend the advanced reporting functionality in this version.
+Though the Analytics module provides an API, it is used specifically to interchange data with MBI. Magento does not recommend you extend the advanced reporting functionality in this version.
```
Comment (keharper) [inline src/guides/v2.3/advanced-reporting/overview.md:30]:
```suggestion
Though the Analytics module provides an API, it is used specifically to interchange data with MBI. Magento does not recommend extending the advanced reporting functionality.
```

File: src/guides/v2.3/advanced-reporting/overview.md (hunk 1 above)
Comment (keharper) [inline src/guides/v2.3/advanced-reporting/overview.md:8]: Global in this doc: remove the `{:target="_blank"}` directive.

**Comment by erikmarr** [comment]
running tests

**Pull request merged**